_pages/team.md

```diff
@@ -1,16 +1,58 @@
 ---
 permalink: /team/
 title: Team 4 All
+feature_row:
+  - image_path: https://projectvikram.github.io/restricted_assets/images/ac_photo.jpg
+    alt: "Abhishek Choudhary (avatar image)"
+    title: "Abhishek Choudhary"
+    excerpt: "Founding Architect"
+    url: "/team/#Abhishek-guriqbal-Choudhary"
+    btn_class: "btn--primary"
+    btn_label: "Profile"
+  - image_path: https://projectvikram.github.io/restricted_assets/images/dr_srija.jpg
+    alt: "Dr Srija Katta (avatar image)"
+    title: "Dr Srija Katta"
+    excerpt: "Technology Evangelist-Pharma & Convenor"
+    url: "/team/#dr-srija-katta"
+    btn_class: "btn--primary"
+    btn_label: "Profile"
 ---
 
-# Team The Project Vikram Journal
+# Team Project Vikram
 
-<img src="https://media-exp1.licdn.com/dms/image/C5603AQGMrVEs2A0qgw/profile-displayphoto-shrink_100_100/0?e=1600905600&v=beta&t=wRz9E9oDLQMAfEWeTirKwD7d5wRn4vWIKB7LROaA1BE" width="300" height="300">
+{% include feature_row %}
+
+<img src="https://projectvikram.github.io/restricted_assets/images/ac_photo.jpg" alttext="Abhishek Choudhary (avatar image)" width="300" height="300">
```

Comment:
Has this pull request been raised on restricted_assets?

Reply:
yes, below is the link for the same

https://github.com/projectvikram/restricted_assets/pulls

Reply:
The posted link should have been https://github.com/projectvikram/restricted_assets/pull/1 (specific pull req #). Linking to "pulls" will not be manageable in high volumes.

Also refer https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue and related documents

Reply:
Thank you, I shall refer